routes/api.php

```diff
@@ -55,17 +52,10 @@
 // Delete Photos
 Route::delete('/photos/delete', 'ApiPhotosController@deleteImage');
 
-// Tag Litter to Photos
-Route::post('/photos/update', 'ApiPhotosController@dynamicUpdate')
-    ->middleware('auth:api');
-
 // Check for any photos uploaded on web
 Route::get('/check-web-photos', 'ApiPhotosController@check')
     ->middleware('auth:api');
 
-Route::post('/add-tags-to-web-img', 'ApiPhotosController@confirm')
-    ->middleware('auth:api');
-
```

Comment:
These are definitely the correct routes to delete? I need to double check this on mobile

Reply:
I asked Anoop if this route was ever used in the app, and it wasn't. But yeah, better double-check this anyway.